modules/basics/src/main/java/com/opengamma/strata/basics/market/MarketData.java

```diff
@@ -60,12 +61,12 @@ public static MarketData of(
 
   //-------------------------------------------------------------------------
   /**
-   * Checks if this set of data contains a value for the specified key.
+   * Returns a box containing values for the specified ID if available.
```

Comment:
"box" has a different meaning in other parts of the MD system. `ReferenceData` uses:

    * Finds the reference data value associated with the specified identifier.
    * <p>
    * If this reference data instance contains the identifier, the value will be returned.
    * Otherwise, an empty optional will be returned.